### PR TITLE
STDTC-61:<SearchAndSortPane> doesn't display results for the first page with pagination

### PR DESCRIPTION
**Pull request by Javokhir12**
## Purpose

- When implemented with pagination, `SearchAndSortPane` component doesn't display results for the first page.
- Refs: https://issues.folio.org/browse/STDTC-61
- See the issue below:

https://user-images.githubusercontent.com/89512426/165698878-331ed5d8-c4a6-4283-be10-10a577c12f1c.mp4

## Approach

- When `<SearchAndSortPane>` component rendered with `pagingType: 'prev-next'`, user can't go back to the first page after he/she navigates to next pages. The reason is that `onNeedMore` handler in `SearchResults` only calls `mutator` function when `index` is  a `truthy` value. But, with `pagination` initial result `offset` will be `0` which is `falsy`.
- Add check for `index`
- Fix `eslint` issues.
- Remove unnecessary `characters` from `FullScreenForm`. See screenshot below:

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

### CODE REVIEW

**Review by mariia-aloshyna — APPROVED** [review]
Thanks for also fixing the unnecessary characters on FullScreenForm, we were just discussing this issue the other day and I haven't even created a bug yet.

File: test/helpers/Harness.js [hunk 1]
```diff
@@ -55,17 +55,17 @@ export function Harness({
...
+            i: chunks => <i>{chunks}</i>,
+            em: chunks => <em>{chunks}</em>,
+            strong: chunks => <strong>{chunks}</strong>,
+            span: chunks => <span>{chunks}</span>,
+            div: chunks => <div>{chunks}</div>,
+            p: chunks => <p>{chunks}</p>,
+            ul: chunks => <ul>{chunks}</ul>,
+            ol: chunks => <ol>{chunks}</ol>,
+            li: chunks => <li>{chunks}</li>,
+            code: chunks => <code>{chunks}</code>,
```
Comment (vsoroka07) [inline test/helpers/Harness.js:68]: Let's use `map` here.

```
const tags = ['b', 'i', ...].map(tag => <tag>{chunks}</tag>);
```

File: test/helpers/hasClassBeginningWith.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { computed } from '@bigtest/interactor';
 
 export function hasClassBeginningWith(selector, className) {
-  return computed(function () {
+  return computed(function checkHasClassBeginningWith() {
```
Comment (vsoroka07) [inline test/helpers/hasClassBeginningWith.js:4]: Is this a common practise to name such functions? IMO it does not bring much value as the function `hasClassBeginningWith` has already described that.

Comment (Javokhir12) [reply test/helpers/hasClassBeginningWith.js:4]: @vsoroka07 it is just to satisfy linter.

File: test/helpers/Harness.js (hunk 1 above)
Comment (Javokhir12) [reply test/helpers/Harness.js]: @vsoroka07 Updated.

**Review by ViktorSoroka07 — APPROVED** [review]

**Pull request merged**